Userland/Libraries/LibGfx/JPEGLoader.cpp

```diff
@@ -787,6 +798,50 @@ static ErrorOr<void> read_icc_profile(SeekableStream& stream, JPEGLoadingContext
     return {};
 }
 
+static ErrorOr<void> read_colour_encoding(SeekableStream& stream, [[maybe_unused]] JPEGLoadingContext& context, int bytes_to_read)
+{
+    // The App 14 segment is application specific in the first JPEG standard.
+    // However, the Adobe implementation is globally accepted and the value of the color transform
+    // was latter standardized as a JPEG-1 extension.
+
+    // For the structure of the App 14 segment, see:
+    // https://www.pdfa.org/norm-refs/5116.DCT_Filter.pdf
+    // 18 Adobe Application-Specific JPEG Marker
+
+    // For the value of color_transform, see:
+    // https://www.itu.int/rec/dologin_pub.asp?lang=e&id=T-REC-T.872-201206-I!!PDF-E&type=items
+    // 6.5.3 - APP14 marker segment for colour encoding
+
+    if (bytes_to_read < 6)
+        return Error::from_string_literal("App14 segment too small");
+
+    [[maybe_unused]] auto const version = TRY(stream.read_value<u8>());
+    [[maybe_unused]] u16 const flag0 = static_cast<u16>(TRY(stream.read_value<u8>())) << 8 | TRY(stream.read_value<u8>());
+    [[maybe_unused]] u16 const flag1 = static_cast<u16>(TRY(stream.read_value<u8>())) << 8 | TRY(stream.read_value<u8>());
```

Comment:
same

Reply:
…oh, I suppose that doesn't doesn't work because of unaligned data. Maybe put a comment saying `// Can't use BigEndian<u16> because the data is unaligned`?

Reply:
Hmm, I guess that `BigEndian<u16>` should also work. My tests aren't really conclusive as flags are always zeroed in all my images.
So let's yeet those nasty `static_cast` and if there is a world where we care about these flags, we will figure out then